### DISCUSSION TITLE
ScriptReference/Build.NamedBuildTarget

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#136] [Notes]
# ScriptReference/Build.NamedBuildTarget

https://docs.unity3d.com/ScriptReference/Build.NamedBuildTarget.html

**Comment by vertxxyz** [comment]
To create a `NamedBuildTarget` from a [`BuildTargetGroup`](https://docs.unity3d.com/ScriptReference/BuildTargetGroup.html), use `NamedBuildTarget.FromBuildTargetGroup(buildTargetGroup)`.
